src/style/app-fixed.module.css

```
@@ -9319,8 +9317,32 @@ form {
+.drawerHeader {
+  padding-right: 50px;
+}
+
+.sidebarText {
+  white-space: nowrap;
+  overflow: hidden;
+  color: #000;
+}
+.expandedDrawer {
+}
+/* ----------------------------------------------------- */
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)